File: code-blog.API/Models/Post.cs
using System;

namespace code_blog.API.Models
{
    public class Post
    {
        public int PostId { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string Author { get; set; }
        public DateTime PublicationDate {get; set;}
        public DateTime UpdatedDate {get; set;}
        public int BlogId { get; set; }
        public string Tags { get; set; }
        public string photoUrl { get; set; }
    }
}

File: code-blog-client/src/app/models/post.ts
export interface Post {
    postId: number;
    title: string;
    content: string;
    author: string;
    updatedDate: Date;
    publicationDate: Date;
    blogId: number;
    tags?: string;
    photoUrl: string;
}

export interface IPostToAddORUpdate{
    postId: number;
    title: string;
    content: string;
    author: string;
    tags?: string;
    photoUrl: string;
}

export interface Pagination {
    currentPage: number;
    itemsPerPage: number;
    totalItems: number;
    totalPages: number;
}

export class PaginatedPosts<T> {
    posts: T;
    pagination: Pagination;
}
